Sources/Cells/BasicTableViewCell/BasicTableViewCell.swift

```diff
@@ -89,7 +89,7 @@ open class BasicTableViewCell: UITableViewCell {
             stackViewToDetailLabelConstraint.isActive = false
         }
 
-        if viewModel.hasChevron {
+        if viewModel.hasChevron == true {
```

Comment:
I think `== true` is not needed here.

Reply:
Right, leftovers from when it was optional 👍